atlas/src/components/VideoBestMatch/VideoBestMatch.style.tsx

```diff
@@ -8,8 +8,12 @@ export const Container = styled.div`
 
 export const Content = styled.div`
   display: grid;
-  grid-template-columns: 650px 1fr;
+  grid-template-columns: minmax(50%, 650px) 1fr;
```

Comment:
This isn't related to this piece of code. But the best match video still suffers from the problem of changing the image aspect ratio when resizing. Also take a look at how it looks like at `500px`, this needs improvement:

<img width="238" alt="Screenshot 2020-10-28 at 12 43 39 PM" src="https://user-images.githubusercontent.com/12646744/97431825-40636800-191b-11eb-8894-7844542fc0ed.png">

Reply:
Didn't catch that, thanks. I've made some changes and now it should look better. The trick to preserve the aspect ratio I've described above is used here as well, again happy to chance if a better solution comes up.